source/_patterns/01-atoms/badge/demo/badges.yml

```diff
@@ -0,0 +1,109 @@
+badge_helper:
+  card_title: Badges
+  card_text: "
+             <p><code>badge_element</code> String: span|a choose the element type for your badge. If you use an link element make sure to enter a value for badge_link</p>
```

Comment:
Maybe just include this in the HTML of the demo? Does it really need to be in the yaml?